### ISSUE TITLE
[BUG] Antialias not working 

### ISSUE THREAD

**Issue by Xyotic** [mono/SkiaSharp#1136]
**Description**

Antialias for SKPaint does not work since SkiaSharp Version 1.68.1 and up
Works as expected with SkiaSharp Version 1.68.0

**Code**
```csharp
       SKPaint Paint { get; set; } = new SKPaint
        {
            Color = SKColors.Black,
            Style = SKPaintStyle.Stroke,
            StrokeCap = SKStrokeCap.Round,
            StrokeJoin = SKStrokeJoin.Round,
            StrokeWidth = 1,
            IsAntialias = true,
            FilterQuality = SKFilterQuality.High 
        };
```

**Expected Behavior**
Antialias is applied to SKPaint

**Actual Behavior**
IsAntialias has no effect

**Basic Information**

- Version with issue:  1.68.1 and 1.68.1.1
- Last known good version:  1.68.0
- IDE:  Visual Studio 2019 v16.4.3

- Platform Target Frameworks:  Tested against the following platforms:
  - Android: 9.0 API-Level 21
  - UWP:  1809 (10.0; Build 17763)
  - ( iOS:  Seems to work fine )

**Screenshots**
SkiaSharp 1.68.1 and up
![SkiaSharp Bug](https://user-images.githubusercontent.com/8405059/74137866-dec66680-4bf0-11ea-9d1f-d0691769d24f.PNG)

SkiaSharp 1.68.0
![SkiaSharp NoBug](https://user-images.githubusercontent.com/8405059/74138840-96a84380-4bf2-11ea-9f6e-64ad3ee23a0f.PNG)

**Comment by mattleibow** [comment]
Could you submit a mini repro? I haven't seen this.

Also, is this by any chance a GL view? I know we had an issue with those using a too-high sample count and then SkiaSharp just bailed and did nothing. That was fixed, and is in the latest 1.68.2 previews. Maybe give those a go and see if that fixes things.

**Comment by Xyotic** [comment]
> 
> 
> Could you submit a mini repro? I haven't seen this.
> 
> Also, is this by any chance a GL view? I know we had an issue with those using a too-high sample count and then SkiaSharp just bailed and did nothing. That was fixed, and is in the latest 1.68.2 previews. Maybe give those a go and see if that fixes things.

You are right. I'm using a GL View and it  seems to be fixed in 1.68.2-preview.29

**Comment by mattleibow** [comment]
@Xyotic, awesome!

Let me link those things for me later on in life... #1098 #1060 #1087  #1053 #1086
